runtime/xamarin/runtime.h

```diff
@@ -109,6 +115,7 @@ struct AssemblyLocations {
 };
 
 void xamarin_initialize ();
+void xamarin_initialize_embedded (); /* STABLE */
```

Comment:
It means someone outside of XI/XM may use this method (in this case the embeddinator), so it shouldn't be removed.